radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java

```diff
@@ -633,6 +633,7 @@ private void doStore(
 			if (transaction != null) {
 				transaction.abort();
 			}
+			log.warn("Error while storing atom: ", e);
```

Comment:
These should be logged by exception handling of caller

Reply:
Removed logging in this place